cpp/include/cudf/strings/detail/convert/int_to_string.cuh

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021, NVIDIA CORPORATION.
+ * Copyright (c) 2022, NVIDIA CORPORATION.
```

Comment:
Should the years prior to 2022 be credited too?

Reply:
Yes, I think that makes sense.